1e/README.md

```diff
@@ -0,0 +1,50 @@
+# 1E
+
+## Overview
+
+The 1E Datadog Integration is an API-based integration that collects metrics from 1E products and forward them to your Datadog account. You can use these metrics to take advantage of the out-of-the-box dashboard or you can create your own visualizations.
+
+## Setup
+
+### Configuration
+
+Getting set up with the 1E Datadog integration is as simple as emailing a Datadog API key and region to the 1E Support team. There's no further configuration required.
+
+1. Go to your [Datadog Integrations page][1] and click on the 1E tile.
+
+2. Click the **Configuration** tab and click **Install Integration** at the bottom.
+
+3. Go to your [Datadog APIs page][2] and create an API key.
+
+4. Determine your Datadog API region by looking at your browser's address bar:
```

Comment:
Datadog API Region should be changed to Datadog Site as this is how we identify regions
```suggestion
4. Determine your Datadog site by looking at your browser's address bar:
```

Reply:
I recommend hyperlinking our datadog site doc: https://docs.datadoghq.com/getting_started/site/